### ISSUE TITLE
strokeDashArray affects markers

### ISSUE THREAD

**Issue by prostolyubo** [observablehq/plot#2131]
## Current State
Certain settings on `strokeDasharray` property may break the markers. Here for instance instead of an arrow on the line we have a tick:
<img width="301" alt="image" src="https://github.com/user-attachments/assets/3a140b79-8dd8-4734-a515-6e60ae81687d">

## Repro
```
Plot.plot({
  marks: [
    Plot.lineY([[0,5],[5,2],[10,0]], { 
      x: (d) => d[0], 
      y : (d) => d[1],
      strokeDasharray: "1,10",
      strokeWidth:3,
      markerMid: "arrow"
    })
  ]
})
```

## Expected State
`strokeDasharray` should not affect markers, or there should be a separate property to control it:
<img width="294" alt="image" src="https://github.com/user-attachments/assets/bdfe2f91-f087-4199-a1d2-649355ecb313">

**Comment by mbostock** [comment]
Yes, this looks like a bug. The `marker` element is inheriting the `stroke-dasharray` because it’s within the `g` of the line mark. We could either move it up so it doesn’t inherit, or set the `stroke-dasharray` attribute on the marker explicitly to `none`. It seems reasonable that the marker shouldn’t inherit dashed strokes.

**Comment by prostolyubo** [comment]
Thanks for confirmation and pointing me into the right direction. Seems like an easy fix for me to figure it out. I'll make a PR if I fix it.
